src/store/index.ts

```diff
@@ -0,0 +1,87 @@
+import Vue from 'vue';
+import Vuex, { Store } from 'vuex';
+import { createDirectStore } from 'direct-vuex';
+
+import api from '@/api';
+import { WorkspaceSpec } from '@/types';
+
+Vue.use(Vuex);
+
+interface State {
+  workspaces: string[];
+  currentWorkspace: WorkspaceSpec | null;
+}
+
+const {
+  store,
+  rootActionContext,
+  moduleActionContext,
+  rootGetterContext,
+  moduleGetterContext,
+} = createDirectStore({
+  state: {
+    workspaces: [],
+    currentWorkspace: null,
+  } as State,
+  getters: {
+    nodeTables(state: State): string[] {
+      if (state.currentWorkspace !== null && state.currentWorkspace.nodeTables) {
+        return state.currentWorkspace.nodeTables;
+      }
+      return [];
+    },
+    edgeTables(state: State): string[] {
```

Comment:
I want to get us in the habit of placing blank lines between multiline code blobs inside of object definitions. It aids readability immensely. This block of code is a great example of where it would help.

Reply:
4346a51